plonky2x/core/src/frontend/eth/mpt/rlc.rs

```diff
@@ -35,7 +35,7 @@ impl<L: PlonkParameters<D>, const D: usize> CircuitBuilder<L, D> {
         b_offset: Variable,
         len: Variable,
     ) {
-        // TODO: implement
+        // TODO: Need to constrain generator result.
```

Comment:
@puma314 Should we constrain this now?

Reply:
No that's fine, no one is going to use this file